solid/lib/WellKnown/OpenIdConfigurationHandler.php

```diff
@@ -117,6 +117,7 @@ private function respond($response) {
 			}
 		}
 		$result->setStatus($statusCode);
+		$result->addHeader('Access-Control-Allow-Origin', '*');
```

Comment:
I'm not sure about the CORS headers, maybe @ylebre has some opinions?

Reply:
This is fine - the rest of the system makes use of the middleware, but the wellknown doesn't so this needs to be in here.